packages/nimble-components/src/table-column/base/models/column-validator.ts

```diff
@@ -34,10 +34,10 @@ export class ColumnValidator<
         } else {
             this.untrack(name);
         }
-        this.updateColumnInternalsFlag();
+        this.updateIsColumnValidFlag();
```

Comment:
Is there still value in having this private method (it seems to me that the value even before your change was questionable)?

Reply:
That's a good point. It doesn't seem very valuable, so I'll get rid of it.